genai-perf/genai_perf/subcommand/analyze.py

```diff
@@ -151,118 +139,31 @@ def sweep(self) -> None:
         for count, objectives in enumerate(
             self._sweep_objective_generator.get_objectives()
         ):
-            #
-            # Create GAP/PA Configs
-            genai_perf_config = GenAIPerfConfig(
-                config=self._config,
-                model_objective_parameters=objectives,
-            )
-
-            perf_analyzer_config = PerfAnalyzerConfig(
-                config=self._config,
-                model_objective_parameters=objectives,
-                extra_args=self._extra_args,
-            )
-
-            #
-            # Check if this configuration has already been profiled (is in the checkpoint)
```

Comment:
Is all of this now duplicated code with the merger? Is that why this can safely be removed?

Reply:
discussed offline. It was moved as part of the refactor to subcommand.py.